plot_krun_results.py

```diff
@@ -452,6 +460,10 @@ def create_cli_parser():
                               "'start,end'. Samples start from 0. e.g. "
                               "'-x 100,130' will show samples in the range "
                               "100 to 130."))
+    parser.add_argument('--single-exec', '-e',
+                        type=int,
+                        help=("Emit only a single execution. "
+                              "e.g. '-e 0' emits only the first."))
```

Comment:
If we want to call them something else we should update all the variable names and the graph titles also.